integration-tests/integration/roles.spec.js

```diff
@@ -104,4 +105,30 @@ context('Roles', () => {
     const roleDetails = RoleDetailsPage.verifyOnPage('Auth Group Manager')
     roleDetails.adminTypes().should('have.length', 2)
   })
+
+  it('should allow change role name', () => {
+    cy.task('stubSignIn', { roles: [{ roleCode: 'MAINTAIN_OAUTH_USERS' }, { roleCode: 'ROLES_ADMIN' }] })
+    cy.signIn()
+
+    cy.task('stubAllRoles', {})
+    cy.task('stubRoleDetails', {})
+    cy.visit('/manage-roles/AUTH_GROUP_MANAGER')
+
+    const roleDetails = RoleDetailsPage.verifyOnPage('Auth Group Manager')
+    roleDetails.changeRoleName()
+
+    cy.task('stubAuthChangeRoleName')
+    cy.task('stubRoleDetails', roleDetailsAfterRoleNameChange)
+    const roleNameChange = RoleNameChangePage.verifyOnPage()
+    roleNameChange.changeName('Name Change')
+
+    RoleDetailsPage.verifyOnPage('New Role Name')
```

Comment:
not sure this is doing anything - since you've stubbed a different value anyway in `roleDetailsAfterRoleNameChange`.  Would be better to verify that you've called auth with the correct parameters e.g. see `verifyAuthUserChangeEmail` task

Reply:
Will update in a separate PR